Some suggestions (#99)
panzerdp/dmitripavlutin.com: posts/172-typescript-function-type/index.md

```
[Open the demo.](https://www.typescriptlang.org/play?#code/C4TwDgpgBAygrgWygXigCgIYC4oDtEBGEATgDRQE74JHECUKAfHoSQFBsDGA9rgM7AofRDAAW3YsBzwkqAGZxcnYAEte6DAwDeUYhGBxiuKBigBfKAHpLUAPIBpLrwFCR4ySWmIUUBUtXqaNq6+obGAAzmUFY2DkA)

> `sumShort` and `sumShorter` have fewer parameters than the type `Sum`, but are still of type `Sum`.  

Reviewer: ```suggestion
`sumShort` and `sumShorter` have fewer parameters than the type `Sum` but are still of type `Sum`.  
```

## 3. TypeScript method type

> A [method](/javascript-method/) is a function that exists and is executed in the context of an object. Method types in TypeScript have to exist within of the object type.  

Reviewer: ```suggestion
A [method](/javascript-method/) is a function that exists and is executed in the context of an object. Method types in TypeScript have to exist within the object type.  
```

## 4. TypeScript function interface

Another interesting way of writing a function type is to use a TypeScript function interface. It's also called function call signature (too scientific for me). 

> The function interface looks a lot like to an object interface with a method. But the function interface *doesn't have the method name written*:  

Reviewer: ```suggestion
The function interface looks similar to an object interface with a method. But the function interface *doesn't have the method name written*:  
```

```
[Open the demo.](https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgMoFcC2B1YYAWAIhAM4JTAAOYwA9iMgN4BQyyAFHAFzIhYBG0ADTJ+PPpkFQAlOIHRWyACalyVGvR4kwFEAHNmAX2bME9bchJYeGHHiKqK1OgwC8yGOhAINITiP5pJkUoCDB0KAY4ZABqUSNmK0wAOhUyJ19kdwByAEEPLx8XZAI4MEssEhKAd1peeSgSbKA)

`SumWithDescription`, in addition to being a function, also has a `description` property of type `string`.  

<ImgShadow>
![TypeScript function interface](./images/typescript-function-interface-2.png)
</ImgShadow>

## 5. Conclusion

> Writing a function type is quite simple:

Reviewer: ```suggestion
Writing a function type is simple:
```
